torch/onnx.py

```diff
@@ -160,5 +161,30 @@ def _at(self, opname, *args, **kwargs):
     return self.op("ATen", *args, operator_s=opname, **kwargs)
 
 
+def _constant(self, value, dims, type, *args, **kwargs):
+    assert isinstance(value, numbers.Number)
+    assert type is not None
+    type = type.lower()
+    if type == "char":
```

Comment:
It would be nice if we could reuse some existing PyTorch code to do this if statement, but it doesn't look like this exists. Perhaps revisit this code when PyTorch gets 'dtype'

Reply:
...and I see that we depend on how ScalarType (C++ side) serializes into string, so I suppose there is no choice here.